install/kubernetes/cilium/templates/spire/server/statefulset.yaml

```diff
@@ -51,9 +51,21 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- if gt (len .Values.authentication.mutual.spire.install.server.initContainers) 0 }}
       initContainers:
+      {{- if gt (len .Values.authentication.mutual.spire.install.server.initContainers) 0 }}
         {{- toYaml .Values.authentication.mutual.spire.install.server.initContainers | nindent 8 }}
+      {{- else }}
+      - name: chown-data
```

Comment:
Can we add a comment here and in the script to remove this once Cilium 1.18 is EOL? Since I understand this is only needed once after upgrading from Cilium 1.18?